### ISSUE TITLE
Package/embed fonts

### ISSUE THREAD

**Issue by taptapdan** [PERTS/perts-ui#13]
@chris-perts:

> I'd like a font solution that doesn't depend on additional network calls. Maybe worth filing an issue for later? I'm guessing all these css files and woff files could be dropped into assets without too much work.

- benefit of cdn based web fonts, not reloading if already cached
- benefit of cdn, resource will load faster since cdn serve might be closer
- fonts.css @import method is blocking? if we don't package our fonts, make sure the method of inclusion isn't blocking
- https://www.keycdn.com/blog/web-font-performance

**Comment by chris-perts** [comment]
Yeah, those are good points. My judgement is being affected by the fact that the jquery cdn had issues two weeks ago.

Considering that our apps are on google, and these fonts are on google, it wouldn't seem to be increasing our risk exposure to continue with cdn-based fonts. Closing.

**Comment by taptapdan** [comment]
I believe I read in the article above, or perhaps another one I looked at, that there might be a way to specify fallback locations for fonts. I'd like to at least icebox this to come back to do to research on if we can fall back to a font that we lazy load from our own hosting: to address the issue you mention.

**Comment by taptapdan** [comment]
Sorry, I clicked the wrong comment button. If you don't agree with my comment, go ahead and reclose! :)
